api.planx.uk/modules/file/service/getFile.ts

```diff
@@ -11,7 +11,7 @@ export const getFileFromS3 = async (fileId: string) => {
   const file = await s3.getObject(params).promise();
 
   return {
-    body: file.Body,
+    body: file.Body as Buffer,
```

Comment:
This should be `S3.Body` which is just `Buffer | Uint8Array | Blob | string | Readable` but this was causing Vultr to run out of heap memory when compiling with `tsc`. It was going from 350mb to over 1gb just for this change - presumably it was importing and compiling a ton of additional AWS types?